packages/js/src/ui/context/InboxContext.tsx

```diff
@@ -88,6 +91,17 @@ export const InboxProvider = (props: InboxProviderProps) => {
     setFilter((old) => ({ ...old, tags: firstTab?.value ?? [] }));
   });
 
+  useNovuEvent({
+    event: 'session.initialize.resolved',
+    eventHandler: ({ data }) => {
```

Comment:
Do we have strong types for this?

Reply:
Yes, it auto infers the correct type on the basis of the event name